sentry_sdk/scope.py

```diff
@@ -1546,6 +1551,13 @@ def __repr__(self):
             self._type,
         )
 
+    @property
+    def flags(self):
+        # type: () -> FlagBuffer
+        if self._flags is None:
+            self._flags = FlagBuffer(capacity=100)
```

Comment:
should this value (100) be a constant that's imported/exported somewhere?

Reply:
Agreed.  I'm not sure how to do it at present but will find out.

Reply:
Perfect thanks :D